CockpitPlugin/src/main/java/com/polidea/cockpitplugin/generator/ReleaseCockpitGenerator.kt

```diff
@@ -23,7 +26,52 @@ class ReleaseCockpitGenerator : BaseCockpitGenerator() {
 
     internal fun createGetterMethodSpecForParam(param: CockpitParam<*>): MethodSpec {
         return createGetterMethodSpecForParamAndConfigurator(param) { builder ->
-            builder.addStatement("return ${createWrappedValueForParam(param)}")
+            val value = param.value
+            when (value) {
+                is List<*> -> builder.addStatement(createArrayListStatement(value as ArrayList<*>).toString(), arraysClassName)
+                else -> builder.addStatement("return ${createWrappedValueForParamValue(param.value)}")
+            }
+        }
+    }
+
+    private fun createArrayListStatement(value: ArrayList<*>): StringBuilder {
```

Comment:
You can reduce entire body to something like this:
`private fun createArrayListStatement(value: ArrayList<Any>) =
            value.map { createWrappedValueForParamValue(it) }.joinToString(separator = ", ", prefix = "return new ArrayList<>(\$T.asList(", postfix = "))")`